.travis.yml

```diff
@@ -41,6 +43,11 @@ script:
   - cd importers/openaddresses && cargo check
   - cargo test
   - cd ../..
-  - cd tools && cargo check
+  # Check for deduplicator
+  - cd deduplicator && cargo check
   - cargo test
   - cd ..
+  # Run OSM importer and a complete deduplication
+  - wget https://download.geofabrik.de/europe/luxembourg-latest.osm.pbf
+  - (cd importers/osm && cargo run --release -- ../../luxembourg-latest.osm.pbf)
```

Comment:
Why the parens?

Reply:
It helps by making the effects of `cd` only last for as long as the scope of the parenthesis (so I don't have to write `cd ..`, this is convenient as I run a single command)

Apparently this is formally due to bash executing the commands in a "subshell environment", from man page ([thx](https://unix.stackexchange.com/questions/26063/how-are-parentheses-interpreted-at-the-command-line)):

```
(list) list is executed in a subshell  environment  (see  COMMAND  EXECUTION  ENVIRONMENT  below).   Variable
       assignments and builtin commands that affect the shell's environment do not remain in effect after the
       command completes.  The return status is the exit status of list.
```